src/scss/layout/_cta.scss

```diff
@@ -8,3 +10,31 @@
   grid-template-columns: minmax(min-content, var(--page));
   justify-content: center;
 }
+
+.hire-us-cta {
+  --section-link-justify: start;
+  --list-padding: var(--gutter-plus);
+  --marker-color: var(--text);
+
+  background-image: url('/assets/media/transition-bg.png');
+  background-repeat: round;
```

Comment:
The result here is a lot of stretching on the image - which would probably be fine if we removed the text from it. I think it could also work to bg-position `center`, and bg-size `cover`.